tools/congestion-model/src/strategy/mod.rs

```diff
@@ -9,17 +9,29 @@ mod global_tx_stop;
 mod no_queues;
 mod simple_backpressure;
 
+// The stats writer can be used to dump stats into a CSV file.
+// In the strategy implementation you should write the header in the init method
+// and the individual values in the compute_chunk method. Please use the
+// write_field. The model will take care of writing the record terminator.
+pub type StatsWriter = Option<Box<csv::Writer<std::fs::File>>>;
+
```

Comment:
Is it necessary to do leak this evaluation code into strategies? It completely breaks the original architecture, where strategies only care about handling congestion and nothing else.

I would prefer to have all evaluation code in the `.src/evaluation` folder and do it independent of specific strategies. The model is aware of all queues the strategy uses, so it should have all the required information.
If we do it this way, the output will also be standardized across all strategies and hence it's much easier to compare them.